Tests/Recurly/Purchase_Test.php

```diff
@@ -93,6 +96,34 @@ public function testAuthorizePurchase() {
     $this->assertNull($collection->charge_invoice->uuid);
   }
 
+  public function testAuthorizeAndCapturePurchase() {
+    $purchase = $this->mockPurchase();
+    $purchase->account->email = 'benjamin.dumonde@example.com';
+    $purchase->account->billing_info->external_hpp_type = 'adyen';
+    $authorizeCollection = Recurly_Purchase::authorize($purchase, $this->client);
+
+    $transactionUUID = $authorizeCollection->charge_invoice->transactions->current()->uuid;
+
+    $cancelCollection = Recurly_Purchase::capture($transactionUUID, $this->client);
```

Comment:
I think that naming this variable `$invoiceCollection` or `$captureCollection` would make more sense now that we've changed the method from `capture` to `cancel`. Not a huge deal, but it might be confusing down the line.